app/assets/stylesheets/marking_schemes.scss

```diff
@@ -1,17 +1,15 @@
 @import 'common/core';
 
-table {
-  td {
-    padding-bottom: 1em;
-    padding-right: 1em;
-  }
-}
 
 h2 {
   padding-bottom: 1em;
 }
 
-.new-ms, 
+.new-ms,
 .edit-ms {
   padding-top: 2em;
 }
+
+#form_actions {
```

Comment:
Avoid using id selectors<br>Selector `form_actions` should be written in lowercase with hyphens